os/net/ipv6/sicslowpan.c

```diff
@@ -689,6 +686,8 @@ compress_hdr_iphc(void)
   uint8_t tmp, iphc0, iphc1, *next_hdr, *next_nhc;
   int ext_hdr_len;
   struct uip_udp_hdr *udp_buf;
+  struct sicslowpan_addr_context *source_context;
+  struct sicslowpan_addr_context *destination_context;
```

Comment:
Every platform supports C99 now, so declare variables closer to their initialization.